src/malloc_wrapper.cc

```diff
@@ -35,6 +35,7 @@ void gnu_backtrace()
 	void *buffer[BT_BUFSZ];
 	char **strings;
 
```

Comment:
1st function call argument is an uninitialized value [clang-analyzer-core.CallAndMessage]